src/unity/ripple-libpp.cpp

```diff
@@ -0,0 +1,86 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of ripple-libpp: https://github.com/ximinez/ripple-libpp
+    Copyright (c) 2016 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <BeastConfig.h>
+
+#include <ripple/protocol/impl/AccountID.cpp>
+#include <ripple/protocol/impl/Book.cpp>
+#include <ripple/protocol/impl/BuildInfo.cpp>
+#include <ripple/protocol/impl/ByteOrder.cpp>
+#include <ripple/protocol/impl/digest.cpp>
+#include <ripple/protocol/impl/ErrorCodes.cpp>
+#include <ripple/protocol/impl/Feature.cpp>
+#include <ripple/protocol/impl/HashPrefix.cpp>
+#include <ripple/protocol/impl/Indexes.cpp>
+#include <ripple/protocol/impl/Issue.cpp>
+#include <ripple/protocol/impl/Keylet.cpp>
+#include <ripple/protocol/impl/LedgerFormats.cpp>
+#include <ripple/protocol/impl/PublicKey.cpp>
+#include <ripple/protocol/impl/Quality.cpp>
+#include <ripple/protocol/impl/Rate2.cpp>
+#include <ripple/protocol/impl/SecretKey.cpp>
+#include <ripple/protocol/impl/Seed.cpp>
+#include <ripple/protocol/impl/Serializer.cpp>
+#include <ripple/protocol/impl/SField.cpp>
+#include <ripple/protocol/impl/Sign.cpp>
+#include <ripple/protocol/impl/SOTemplate.cpp>
+#include <ripple/protocol/impl/TER.cpp>
+#include <ripple/protocol/impl/tokens.cpp>
+#include <ripple/protocol/impl/TxFormats.cpp>
+#include <ripple/protocol/impl/UintTypes.cpp>
```

Comment:
A full build from scratch (including building the Makefile) took about 19 seconds for me.  That felt pretty reasonable.